src/components/MoneyReportHeader.tsx

```diff
@@ -899,9 +899,9 @@ function MoneyReportHeader({
         [connectedIntegrationName, styles.noWrap, styles.textStrong, translate],
     );
 
-    const {selectionMode} = useMobileSelectionMode();
+    const selectionMode = useMobileSelectionMode();
```

Comment:
```suggestion
    const isSelectionModeEnabled = useMobileSelectionMode();
```

Since previously we did `selectionMode.isEnabled` and I think it was more informative.